packages/core/src/credentials-provider/DefaultCredentialsProvider.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { CredentialProvider } from './CredentialsProvider';
+import { EnvCredentialProvider } from './DefaultCredentialProvider';
+
+describe('EnvCredentialProvider', () => {
+  it('sholud create new instance', () => {
+    const provider = new EnvCredentialProvider();
+    expect(provider).toBeDefined();
+  });
```

Comment:
It is totally useless check. Don't tell me that you decided to test the `new` operator)